components/ILIAS/Certificate/classes/class.ilObjCertificateSettingsGUI.php

```diff
@@ -82,6 +98,14 @@ public function getAdminTabs(): void
             );
         }
 
+        if ($this->rbac_system->checkAccess('visible,read', $this->object->getRefId())) {
```

Comment:
We still need to check whether the certificates are globally enabled. Otherwise, we should NOT display this tab.

Maybe we should use `ilCertificateActiveValidator` instead?